### PR TITLE
Limit can be configured

### PR DESCRIPTION
**Pull request by Masclins**
This solves #88.

It required changing the structure of `:rows` and adapting the backend tests for it. 

Right now limit configured is used for nothing, and the rows are passed to frontend in the same format as before.

### CODE REVIEW

File: backend/src/api/base.clj [hunk 1]
```diff
@@ -13,11 +13,11 @@
   (player-view/get-game-as-player (persistence/fetch-game game-id) player-id))
 
 (defn play-card-as-player
-  [game-id player index row & target]
+  [game-id player index row-id & target]
```
Comment (kenan-rhoton) [inline backend/src/api/base.clj]: yay, clarity!

File: backend/src/api/player_view_functions.clj [hunk 2]
```diff
@@ -8,13 +8,19 @@
...
+
 (defn get-rows
   "Return the rows as seen by the player"
   [game-state player-id]
-  (mapv #(mapv (fn [card]
-                 (assoc card :owner
-                   (conversions/translate-player game-state (:owner card) player-id)))
-               %)
+  (mapv (fn [row]
+          (get-row-cards (:cards row) game-state player-id))
```
Comment (kenan-rhoton) [inline backend/src/api/player_view_functions.clj]: You are currently eliminating in this step any information other than the contents of `:cards` from a row. Is this what you intend?

Comment (Masclins) [reply backend/src/api/player_view_functions.clj]: It is **right now**.
My only intent with this pull-request is to change the backend so it handles row limits, but frontend will be done in another pull-request (for #90).
So the api should keep giving the same as before to frontend. That's why here it loses information.

File: backend/test/rules/play/next_play.clj [hunk 3]
```diff
@@ -2,18 +2,21 @@
   (:require [expectations.clojure.test :refer :all]
             [rules.play-card :as play-card]))
 
+(def empty-rows
+  [{:cards []} {:cards []} {:cards []}])
```
Comment (kenan-rhoton) [inline backend/test/rules/play/next_play.clj]: I definitely like this.

File: backend/src/api/player_view_functions.clj (hunk 2 above)
Comment (kenan-rhoton) [reply backend/src/api/player_view_functions.clj]: I thought so, but wanted to double-check and comment so we don't forget 🙃

**Pull request merged**